app/src/main/java/com/github/polypoly/app/base/game/Game.kt

```diff
@@ -31,13 +31,14 @@ class Game private constructor(
             owner = null,
             level = PropertyLevel.LEVEL_0,
         ) } }
-    val currentRound: Int = 1
+    var currentRound: Int = 1
 
     /**
      * Go to the next turn
      */
     fun nextTurn() {
         // TODO update the data with the DB
+        ++currentRound
```

Comment:
Maybe we should keep a track of the round number in the db itself, to avoid any player having false round number in local?

Reply:
This is an intended behavior, this data should indeed be synchronised across all the players (as the comment above tried to suggest). Here this pr doesn't aim at doing that, but the goal was instead to lay a solid fondation for the data logic/UI interactions that will be needed in the next sprints. I'll update this pr's doc to reflect this intent better :)